umbraco-cloud/product-upgrades/minor-upgrades.md

```diff
@@ -17,10 +17,15 @@ To enable Automatic Minor Upgrades follow these steps:
     <figure><img src="../.gitbook/assets/image (10).png" alt=""><figcaption><p>Enable Minor Upgrades</p></figcaption></figure>
 
 With automatic upgrades enabled it will be applied to all products on Umbraco Cloud: Umbraco CMS, Umbraco Forms, and Umbraco Deploy.
+Your project does not need to be running the latest minor for automatic upgrades to work. The project will simply be upgraded to the latest minor when it is released.
```

Comment:
as part of our styleguide we have setup so that "simply" or "simple" to not be used in the documentation: https://docs.umbraco.com/welcome/contribute/style-guide#editorializing
```suggestion
Your project does not need to be running the latest minor for automatic upgrades to work. The project will be upgraded to the latest minor when it is released.
```